builder.nix

```diff
@@ -86,6 +86,8 @@ pkgs.stdenv.mkDerivation {
         ) allRequiredPackages
       );
     in ''
+      runHook preConfigure
```

Comment:
It's recommended to include `runHook pre{Phase}` & `runHook post{Phase}` when overriding the phase:
>When overriding a phase, for example installPhase, it is important to start with runHook preInstall and end it with runHook postInstall, otherwise preInstall and postInstall will not be run. Even if you don’t use them directly, it is good practice to do so anyways for downstream users who would want to add a postInstall by overriding your derivation.

https://github.com/NixOS/nixpkgs/blob/e6fd6b9dc098c7f21f5ddf1354a5c5b80f212e1e/doc/stdenv/stdenv.chapter.md?plain=1#L646